app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -49,16 +70,24 @@ define(function (require, exports, module) {
           // default image if displayed
           return new ProfileImage();
         })
+        .fin(function () {
+          if (typeof spinnerEl !== 'undefined') {
+            return self._completeLoadingSpinner(spinnerEl, options && options.spinnerTimeout);
```

Comment:
To handle `undefined` options, our convention is to set `options` to an empty object at the top of the function:

```js
displayAccountProfileImage: function (account, wrapperClass, options) {
  options = options || {};
```

Using ES6 defaults would be awesome, but we haven't jumped on that ship yet.